src/components/InventoryManager/ListingSteps/PricesStep.js

```diff
@@ -0,0 +1,248 @@
+                </Typography>
+              </Typography>
+            </Grid>
+            <Grid container item xs={12} spacing={1}>
+              <Grid item xs={12}>
+                <Typography className={classes.text}>
+                  {`This is what your prices per ${validPrices} will be:`}
+                </Typography>
+              </Grid>
+              {listingRecord['has master units']
+              && (
+              <Grid item xs={12}>
+                <Typography className={classes.labels}>
+                  Standard Price per Master Unit:
+                  <Typography display="inline" className={classes.unitLabel}>
+                    $
+                  </Typography>
+                  <Typography display="inline" className={classes.text}>
+                    {`${Number(standardPrices.masterUnitPrice).toFixed(2) || '0.00'} `}
+                    per Master
+                    {` ${listingRecord['grouped produce type']}`}
+                  </Typography>
+                </Typography>
+              </Grid>
+              )}
+              <Grid item xs={12}>
+                <Typography className={classes.labels}>
+                  Standard Price per Pallet:
+                  <Typography display="inline" className={classes.unitLabel}>
+                    $
+                  </Typography>
+                  <Typography display="inline" className={classes.text}>
+                    {`${Number(standardPrices.palletPrice).toFixed(2) || '0.00'} `}
+                    per Pallet
+                  </Typography>
+                </Typography>
+              </Grid>
+              {listingRecord['has master pallets']
+              && (
+              <Grid item xs={12}>
+                <Typography className={classes.labels}>
+                  Standard Price per Master Pallet:
+                  <Typography display="inline" className={classes.unitLabel}>
+                    $
+                  </Typography>
+                  <Typography display="inline" className={classes.text}>
+                    {`${Number(standardPrices.masterPalletPrice).toFixed(2) || '0.00'} `}
+                    per Master Pallet
+                  </Typography>
+                </Typography>
+              </Grid>
+              )}
+            </Grid>
+          </Grid>
+        </Grid>
+        <Grid item container xs={12}>
+          <Typography variant="h2" component="h2" className={classes.heading}>{questions[1]}</Typography>
+        </Grid>
+        <Grid container item xs={12}>
+          <Grid container spacing={1} item xs={12}>
+            <Grid item xs={12}>
+              <Typography className={classes.subheading} variant="h6" component="h6">
+                Agency Price per Grouped Product Type
+              </Typography>
+            </Grid>
+            <Grid container item xs={12} alignItems="center">
+              <ListingInputField
```

Comment:
Make sure that this listing input field cannot be negative or 0 when you move forward to the next step